starboard/nplb/socket_get_interface_address_test.cc

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#if SB_API_VERSION < 17
```

Comment:
why are we keeping code that is  for SB < 17?

Reply:
Ah sorry forgot to add in a comment -- I kept these tests with the flag because I wasn't sure if we need to posix-ify these tests. There aren't equivalent versions in the posix_compliance version. For tests that have a posix_compliance equivalent, I'm deleting the old files.